app/actions/client.py

```diff
@@ -0,0 +1,725 @@
+import httpx
+import json
+import logging
+import pydantic
+import random
+import re
+import os
+
+import app.actions.utils as utils
+
+from app.actions.configurations import AuthenticateConfig
+from app.services.errors import ConfigurationNotFound
+from app.services.state import IntegrationStateManager
+from app.services.utils import find_config_for_action
+
+from shapely.geometry import GeometryCollection, shape, mapping
+
+from datetime import datetime, timedelta, timezone
+from typing import Optional, List, Set, Tuple
+
+logger = logging.getLogger(__name__)
+state_manager = IntegrationStateManager()
+
+
+DEFAULT_FIRE_LOOKBACK_DAYS = 1
+DEFAULT_GLAD_LOOKBACK_DAYS = 1
```

Comment:
This should be a higher number, probably 10 days.

We might want to have these two `_LOOKBACK_DAYS` be configurable, with some sensible maximums.

Reply:
We can have that variable to apply to both types? or we should ask for both in the config?